bears/python/requirements/PySafetyBear.py

```diff
@@ -0,0 +1,93 @@
+from collections import namedtuple
+import pkg_resources
+import re
+
```

Comment:
Can you link me to any guideline/rule we have for this? I'm ordering them alphabetically here.